src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -17,10 +20,32 @@ public SelectTestFigureOptionListener(DriverManager driverManager) {
 	@Override
 	public void actionPerformed(ActionEvent e) {
 		if(e.getActionCommand().equals("Figure Joe 1")){
+
 			FiguresJoe.figureScript1(driverManager.getCurrentDriver());
+
 		}
-		else{
+		else if(e.getActionCommand().equals("Figure Joe 2")){
+
 			FiguresJoe.figureScript2(driverManager.getCurrentDriver());
+
+		}
+		else if(e.getActionCommand().equals("Rectangle")){
```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.